src/js/_enqueues/admin/site-icon.js

```diff
@@ -1,15 +1,40 @@
-(function($) {
-	var frame;
-
-	function calculateImageSelectOptions ( attachment ) {
-		var realWidth  = attachment.get( 'width' ),
+/**
+ * Handle the site icon setting in options-general.php.
+ *
+ * @since 6.5.0
+ * @output wp-admin/js/site-icon.js
+ */
+
+/* global jQuery, wp */
+
+( function( $ ) {
+	var $chooseButton = $( '#choose-from-library-button' ),
+	$iconPreview = $( '#site-icon-preview' ),
+	$browserIconPreview = $( '#browser-icon-preview' ),
+	$appIconPreview = $( '#app-icon-preview' ),
+	$hiddenDataField = $( '#site_icon_hidden_field' ),
+	$removeButton = $( '#js-remove-site-icon' ),
+	frame;
+
+	/**
+	 * Calculates image selection options based on the attachment dimensions.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param {Object} attachment The attachment object representing the image.
+	 * @return {Object} The image selection options.
+	 */
+	function calculateImageSelectOptions( attachment ) {
+		var realWidth = attachment.get( 'width' ),
 			realHeight = attachment.get( 'height' ),
 			xInit = 512,
 			yInit = 512,
 			ratio = xInit / yInit,
-			xImg  = xInit,
-			yImg  = yInit,
-			x1, y1, imgSelectOptions;
+			xImg = xInit,
+			yImg = yInit,
+			x1,
+			y1,
+			imgSelectOptions;
```

Comment:
```suggestion
			x1, y1, imgSelectOptions;
```

declarations can be grouped on a single line

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-var